runtime/src/lib.rs

```diff
@@ -190,10 +190,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("subsocial-parachain"),
 	impl_name: create_runtime_str!("subsocial-parachain"),
 	authoring_version: 1,
-	spec_version: 44,
+	spec_version: 45,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 9,
+	transaction_version: 10,
```

Comment:
I don't see us changing transactions in this PR. 
Why do we need to increase the number?

Reply:
Because they changed in FRAME. Will add a comparison results to a base branch